### ISSUE TITLE
Make the floating button a transparent pill

### ISSUE THREAD

**Issue by omar-cherif** [mawaqit/android-tv#21]
Now that the remote left and right triggers the menu, we can make the floating button transparent.
So, this button should now become transparent to avoid readability issues on hadith screen and other screens (like phone silence message etc.), navigation, user manual, about us etc...

It becomes [Temporarily] a 'hidden/secret' button for now (giving the impression that a click on the Mawaqit Logo triggers the menu) if the user uses the mouse.

In other words :
- Make the menu button transparent
- Place it above the Mawaqit logo (bottom right)
- Ajdust the shape + size from circle to pill (Same size as the bottom right Mawaqit logo)
